### PR TITLE
Merge & correct the layout of the sidebar

### PR DESCRIPTION
**Pull request by Ajotah98**
I changed the layout to correct the grid problem on the header.

Also, I wasn't aware that the sidebar view also had the same problem for long titles, so I added a display inline that makes a better UI (in my opinion)

### CODE REVIEW

**Comment by lovasoa** [comment]
@Ajotah98 , I think you have a formatter that changed a bunch of files that are not related to the PR. Can you revert the formatting changes and keep only the actual shell component updates?

File: sqlpage/templates/shell.handlebars [hunk 1]
```diff
@@ -147,14 +147,14 @@
...
-            <header id="sqlpage_header"></header>
+            <header id="sqlpage_header">
         {{#if sidebar}}
         <aside class="navbar navbar-vertical navbar-expand-lg" {{#if sidebar-theme}}data-bs-theme="dark" {{/if}}>
             <div class="container-fluid">
                 <button class="navbar-toggler collapsed" type="button" data-bs-target="#sidebar-menu" aria-controls="sidebar-menu" data-bs-toggle="collapse" aria-expanded="false" aria-label="Toggle navigation">
                     <span class="navbar-toggler-icon"></span>
                 </button>
-                <h1 class="navbar-brand navbar-brand-autodark">
+                <h1 class="navbar-brand navbar-brand-autodark" style="display:inline;">
```
Comment (lovasoa) [inline sqlpage/templates/shell.handlebars:157]: can we use a bootstrap class instead of an inline style ?

**Review by lovasoa — CHANGES_REQUESTED** [review]

**Comment by lovasoa** [comment]
additionnally, this seems to break long titles ellipsis

image

**Comment by lovasoa** [comment]
Ok, I think everything is fixed now

**Comment by Ajotah98** [comment]
I apologize!! I couldn't see this until today. 

I'm grateful you managed to repair the broken things, and so sorry for breaking the title.
Thanks!!

**Pull request merged**